packages/collector/test/metrics/app/node_modules/node-fetch/CHANGELOG.md

```diff
@@ -1,266 +0,0 @@
-
```

Comment:
The whole node_modules folder for this application under test was previously under version control. I have removed that and the test runs `npm install` in the applications directory now instead.